libbeat/docs/shared-docker.asciidoc

```diff
@@ -104,24 +104,11 @@ configuration included.
 
 Download this example configuration file as a starting point:
 
-ifndef::apm-server[]
-
 ["source","sh",subs="attributes,callouts"]
 ------------------------------------------------
-curl -L -O https://raw.githubusercontent.com/elastic/beats/{doc-branch}/deploy/docker/{beatname_lc}.docker.yml
+curl -L -O {dockerconfig}
```

Comment:
Looks good! Now you just need to make sure you set {dockerconfig} in your index file.